src/config.ts

```diff
@@ -17,6 +17,7 @@ const proteinsApiServices = {
 
 export type TrackType =
   | 'nightingale-track'
```

Comment:
I can't see this being used anywhere so I think the nightingale-track package and references can be removed.